svthalia/concrexit
# Cache static content for maximum time

Closes #2184 

### Summary
Allows the browser to cache the files in /static for 10 years without revalidation. Information at: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

### How to test
Steps to test the changes you made:
1. View any page in the browser network inspection tab
2. Check that files from /static are cached
